packages/playground/wordpress-builds/build/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM emscripten/emsdk:3.1.24 as emscripten
+FROM emscripten/emsdk:3.1.61 as emscripten
```

Comment:
I bumped this to match the current Emscripten version because my local build could no longer pull everything the older version required.